src/actions.ts

```diff
@@ -290,69 +302,61 @@ export const blur = () => ({
 
 // Key Bindings
 
-type KeyDownHandlers<Cell: Types.CellBase> = {
-  [eventType: string]: KeyDownHandler<Cell>
+type KeyDownHandlers<Cell> = {
+  [eventType: string]: KeyDownHandler<Cell>;
 };
 
 /** @todo handle inactive state? */
-const keyDownHandlers: KeyDownHandlers<*> = {
+const keyDownHandlers: KeyDownHandlers<any> = {
   ArrowUp: go(-1, 0),
   ArrowDown: go(+1, 0),
   ArrowLeft: go(0, -1),
   ArrowRight: go(0, +1),
   Tab: go(0, +1),
-  Enter: edit,
+  Enter: edit as KeyDownHandler<any>,
```

Comment:
Generally concerned about the `as` usage everywhere. Should be done sparingly and with justification (likely in a comment)